### DISCUSSION TITLE
Question: `fh` with predicting T-20 and T+10 steps?

### DISCUSSION THREAD

**Discussion by BradKML** [alan-turing-institute/sktime#3656] [Q&A]
Let's say there is a dataset with daily information, If I want to forecast the last 20 days (seeing how models fit with reality), combined with forecasting the next 10 days (blind forecast), is this syntax correct?
```python
  y_pred = gscv.predict(fh=arange(-2*forecast_count, forecast_count+1))
  fig, _ = plot_series(y[-forecast_count*2:], y_pred, labels=["y", "y_pred"])
```

for the model building process:
```python
  forecast_count = 10
  cv = ExpandingWindowSplitter(
      initial_window=forecast_count*2, # at least 20 data points
      step_length=forecast_count, # doing it in 10 data point chunks
      start_with_window=True,
      fh=list(range(1,forecast_count+1))) # predicting the next data points, fh is list NOT range

  gscv = ForecastingGridSearchCV(
      forecaster=pipe,
      param_grid=[......], # too long to list
      cv=cv, n_jobs=-1,
      return_n_best_forecasters=10)
```

**Comment by aiwalter** [comment]
`list(range(1,10+1))` gives:
`[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]`
So this is only forecasting the next 10 days.

But what you described in the text would be this:
`list(range(-19,10+1))` which is:
`[-19,
 -18,
 -17,
 -16,
 -15,
 -14,
 -13,
 -12,
 -11,
 -10,
 -9,
 -8,
 -7,
 -6,
 -5,
 -4,
 -3,
 -2,
 -1,
 0,
 1,
 2,
 3,
 4,
 5,
 6,
 7,
 8,
 9,
 10]`
This is last 20 days (in-sample prediction) plus next 10 days

**Reply by BradKML** [reply]
Second Question: how can this in-sample + out-of-sample prediction be combined into one single graph against ground truth?
